### ISSUE TITLE
Unify all syntax highlights

### ISSUE THREAD

**Issue by ahmadabdolsaheb** [freeCodeCamp/freeCodeCamp#37166]
It would be optimal to have code-blocks share the same or similar syntax highlighting with the editor based on the background.

All syntax highlights should have at least the contrast ratio of 7.0 between the font and the background
All syntax highlighting could use predefined global variables for consistency.

**Comment by ojeytonwilliams** [comment]
If memory serves, the prism code blocks and the Monaco code blocks mark up code slightly differently.  What I tried to do the first time was make the prism blocks as similar to the editor as possible, but I don't think it's possible to get an exact match that way.  The editor needs changing too (not sure why I didn't!).

Anyway, that's what I remember from when I last looked at this!

**Comment by ahmadabdolsaheb** [comment]
I will look into this a little bit more but from what I remember we could change the color, size, and even the front of Monaco.

**Comment by ojeytonwilliams** [comment]
Yeah, looks like you can change things via [defineTheme](https://microsoft.github.io/monaco-editor/api/modules/monaco.editor.html#definetheme).  We already set some things that way.

Also, if that doesn't work you can always target the classes directly via selectors like

```css
span.mtk7 {
  color: red
}
```

`mtk7` could well be an implementation detail, though, so I'd only go this route as a last resort.

**Comment by moT01** [comment]
Thank you for reporting this issue.

This is a standard message notifying you that the issue has become stale, so we are going to close it.

If you think we're wrong in closing this issue, please request for it to be reopened. Thank you and happy coding. 🚽
